Automattic/wp-calypso
# Fix Jetpack Social Manage Subscription button

## Proposed Changes

* Adds missing constants for Jetpack Social that were causing the lightbox not to show up when the user owns Jetpack Social
* Adds other missing constants for other 2 year plans (FEATURE constants)

## Testing Instructions

* Click on the Calypso live link or checkout this branch and start your local Calypso Green environment
* Using a jurassic ninja or ephemeral site, purchase a Jetpack Social Basic or a Jetpack Social Advanced 2 year plan
* After completing the purchase, make your way back to the pricing page with the site in context (http://jetpack.cloud.localhost:3000/pricing/{your-site})
* Click on the "Manage Subscription" button on the Social pricing card
![image](https://github.com/Automattic/wp-calypso/assets/65001528/15498a48-c7db-4adb-ae4b-c302770a3c74)
* Make sure the lightbox comes up instead of going to checkout. If you have the 2 year basic plan, you should see a "Manage Subscription" button underneath the Basic option, and the option to add Advanced to the cart
![image](https://github.com/Automattic/wp-calypso/assets/65001528/9f529c50-6730-4185-aaec-cf492177d9a7)
![image](https://github.com/Automattic/wp-calypso/assets/65001528/6abde470-09e8-4099-bf4a-e0a327ad8362)
* Make sure both CTA's work
* If you have Jetpack Social Advanced you should see the "Manage Subscription" button under Jetpack Social Advanced in the lightbox
![image](https://github.com/Automattic/wp-calypso/assets/65001528/84a72f6c-2a09-47a7-9bbb-07826d6acecd)
* There is an issue I discovered where Jetpack Social basic will show the option to add it to the cart (or go to checkout) if the user has Jetpack Social Advanced, but throw an error because they already have a better product. This is unrelated to this PR and will fixed elsewhere

## Pre-merge Checklist
